### ISSUE TITLE
Expected Performance?

### ISSUE THREAD

**Issue by mfe7** [maximecb/gym-minigrid#17]
Could anyone provide what type of performance they get on these domains with some of the provided baseline algorithms?

It is probably overkill to maintain something like a leaderboard, but it would be helpful to have some mechanism to confirm that the RL algorithms are behaving as expected and everything is installed correctly. Sometimes plots of reward vs episode number are provided in the readme. In addition, it would be cool to hear about any hyperparameters that were found to be particularly important for good performance.

For at least an order-of-magnitude example based on my installation, without making any changes to the repo, I saw convergence to ~0.9 mean reward in 130,000 timesteps when running this command:
```
python3 /home/mfe/code/gyms/gym-minigrid/pytorch_rl/main.py --env-name MiniGrid-Empty-6x6-v0 --no-vis --num-processes 1 --algo a2c
```
which took about 5mins on my i7-6700 w/ 1060GTX, though I don't know whether the GPU was being used by default.

For the 8x8 empty domain, it took about 45 mins and 1.8M timesteps to converge to ~0.9 reward with this command:
```
python3 /home/mfe/code/gyms/gym-minigrid/pytorch_rl/main.py --env-name MiniGrid-Empty-8x8-v0 --no-vis --num-processes 1 --algo a2c
```

**Comment by maximecb** [comment]
No hyper-parameter search has been done, but the code in https://github.com/lcswillems/pytorch-a2c-ppo uses hyper-parameters that are known to work.
